File: PortalAboutEverything/PortalAboutEverything/Models/BoardGame/BoardGameViewModel.cs
using PortalAboutEverything.Models.BoardGameReview;

namespace PortalAboutEverything.Models.BoardGame
{
    public class BoardGameViewModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string MiniTitle { get; set; }
        public bool HasMainImage { get; set; }
        public bool HasSideImage { get; set; }
        public string Description { get; set; }
        public string? Essence { get; set; }
        public string? Tags { get; set; }
        public double Price { get; set; }
        public long ProductCode { get; set; }
        public bool IsFavoriteForUser { get; set; }
    }
}

File: react/portal-react/src/models/boardGames/BoardGameViewModel.ts
export default interface BoardGameViewModel {
    id: number;
    title: string;
    miniTitle: string;
    hasMainImage: boolean;
    hasSideImage: boolean;
    description: string;
    essence: string;
    tags: string;
    price: number;
    productCode: number;
    isFavoriteForUser: boolean;
}
